recordedit/recordEdit.app.js

```diff
@@ -186,8 +186,7 @@
 
                         // populate defaults
                         angular.forEach($rootScope.reference.columns, function(column) {
-                            // typeof column.default !== "function"
-                            if (column.default) {
+                            if (column.default && typeof column.default !== "function" && !recordEditModel.rows[0][column.name]) {
```

Comment:
A comment would be useful here for why you're including a check for `recordEditModel.rows` here, and maybe also explain the check for function as well.